Curtains/Curtains.Application_tests/UnitTest1.cs

```diff
@@ -0,0 +1,16 @@
+namespace Curtains.Application_tests
```

Comment:
Please change naming to Curtains.Application.Tests
Also move all tests projects to separated folder with naming Tests